app/Http/Controllers/Lessons/LoadLessonController.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App\Http\Controllers\Lessons;
+
+use App\Models\User;
+use App\Models\Course;
+use App\Models\Lesson;
+use App\Http\Controllers\Controller;
+
+class LoadLessonController extends Controller
+{
+    public function __invoke(Course $course, Lesson $lesson)
+    {
+        try {
+            $state = auth()->user()->lessons()->find($lesson->id)->pivot->game_state;
```

Comment:
it might be better to format this one like the save lesson controller :) 

```suggestion
            $state = auth()->user()
                ->lessons()
                ->find($lesson->id)
                ->pivot
                ->game_state;
```